userspace/libsinsp/container.cpp

```diff
@@ -400,6 +403,9 @@ void sinsp_container_manager::dump_containers(sinsp_dumper& dumper)
 		sinsp_evt evt;
 		if(container_to_sinsp_event(container_to_json(*it.second), &evt, it.second->get_tinfo(m_inspector)))
 		{
+			evt.m_pevt->ts = (m_inspector->m_lastevent_ts == 0)
```

Comment:
I suppose we need the same thing for users and groups when dumped to scap-files

Reply:
I agree!